evaluator/evaluator.py

```diff
@@ -189,9 +197,9 @@ async def _vmaas_request_cves(vmaas_request_json):
                                                         vmaas_request_json)
     if vulnerabilities_response_json is not None:
         for cve in vulnerabilities_response_json['cve_list']:
-            playbook_cves.add(cve)
+            playbook_cves.append(Cve(cve["cve"], ",".join(cve["errata"])))
         for cve in vulnerabilities_response_json['manually_fixable_cve_list']:
-            manually_fixable_cves.add(cve)
+            manually_fixable_cves.append(Cve(cve["cve"], ",".join(cve["errata"])))
```

Comment:
This is invalid state, manually fixable CVE never has any errata. Also make sure empty string is never inserted to the DB.